From f132655f458035aa2ed6b9e3a5ae4c04b13c9311 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 28 Apr 2023 14:47:07 -0400
Subject: [PATCH 23/35] test: Make our fake dprintf be a statement.

In a few places we put dprintf() at places where the compiler will get
confused if it isn't a block or a statement.

Obviously, it should be a statement, so this makes it one.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 include/console.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/console.h b/include/console.h
index 8eb4b47..7ac4e11 100644
--- a/include/console.h
+++ b/include/console.h
@@ -106,8 +106,8 @@ extern UINT32 verbose;
 	dprint_(L"%a:%d:%a() " fmt, __FILE__, __LINE__ - 1, __func__, \
 	        ##__VA_ARGS__)
 #else
-#define dprint_(...)
-#define dprint(fmt, ...)
+#define dprint_(...) ({ ; })
+#define dprint(fmt, ...) ({ ; })
 #endif
 
 extern EFI_STATUS EFIAPI vdprint_(const CHAR16 *fmt, const char *file, int line,
-- 
2.39.0

